### PR TITLE
Changed date format to date ISO standard 8601

### LINKED ISSUE #1: Logfile date should be in different order
In order to better sort the logs it would be better to save it as `2016-02-29.crsv.log`.

https://github.com/CanOpener/serverlog/blob/master/serverlog.go#L148

### PR DESCRIPTION
**Pull request by VoyTechnology**
Closes #1

### CODE REVIEW

**Comment by VoyTechnology** [comment]
@CanOpener any updates on this being merged?

File: serverlog.go [hunk 1]
```diff
@@ -145,7 +145,7 @@ type logItem struct {
 
 // listen is the listener which runs in its own gorutine and logs messages
 func listen() {
-	currentLogPath := path.Join(logDir, time.Now().Format("02-01-2006.crsv.log"))
+	currentLogPath := path.Join(logDir, time.Now().Format("2016-02-27.crsv.log"))
```
Comment (CanOpener) [inline serverlog.go]: There is a specific reason i chose the date 02/01/2006
take a look at https://gobyexample.com/time-formatting-parsing

File: serverlog.go [hunk 2]
```diff
@@ -199,7 +199,7 @@ func logFileOverseer() {
 
 		select {
 		case <-newDay:
-			newLogFile := path.Join(logDir, tomorrow.Format("02-01-2006.crsv.log"))
+			newLogFile := path.Join(logDir, tomorrow.Format("2016-02-27.crsv.log"))
```
Comment (CanOpener) [inline serverlog.go]: same as above

**Comment by VoyTechnology** [comment]
@CanOpener I fixed the problem

**Pull request merged**